packages/web3/package.json

```
@@ -0,0 +1,102 @@
+{
+  "name": "@web3modal/web3",
```

```suggestion
  "name": "@chainsafe/web3",
```
As web3 modal repo in Chainsafe we will release it under `chainsafe` org on npm for now, 
and once your actual PR is merged in web3modal they will release it under `web3modal` org

---

Thansk,
As communicated, for now it is published under https://www.npmjs.com/package/web3modal-web3js